insights/core/spec_factory.py

```diff
@@ -142,18 +143,20 @@ def validate_lines(self, results):
         return True
 
     def load(self):
-        # read with universal newlines so all line terminators are converted to
-        # "\n"
-        with open(self.path, 'U') as f:
-            results = []
-            if self.ds:
-                # This should shell out to a grep pipeline
-                results = [l.rstrip("\n") for l in apply_filters(self.ds, f)]
+
+        filters = False
+        if self.ds:
+            filters = "|".join(get_filters(self.ds))
+        if filters:
+            rc, out = subproc.call("grep -E '{0}' {1} ".format(filters, self.path), shell=False, keep_rc=True)
```

Comment:
Are we making a conscious decision to change from fixed string to regex grepping?

Reply:
The idea was to remove the file open/read and separate apply_filters() to improve performance. Reformatting the list of filters and using grep to traverse and filter the lines in the file based on the filters seemed like the simplest and fastest way top do it.

Reply:
Sorry, I should have made it clear that I was referring specifically to your `-E` option.  Historically we've used `-F`, and I was wondering if this is something that we are intentionally changing at this point.